src/components/common/MediaCard.js

```diff
@@ -15,6 +15,8 @@ import DeleteButton from '../space/DeleteButton';
 import ExportButton from '../space/ExportButton';
 import SyncButton from '../space/SyncButton';
 import { MIN_CARD_WIDTH } from '../../config/constants';
+import Text from './Text';
+import './MediaCard.css';
```

Comment:
Instead of using the `.css`, look at the `styles` object. You can add the `fontSize` property to a `cardDescriptionText` key and pass that as a class.